### DISCUSSION TITLE
Leica and Aria Triggering?

### DISCUSSION THREAD

**Discussion by rvmoriarty** [IBEXImagingCommunity/ibex_imaging_knowledge_base#241] [Q&A]
Hi all, 

Has anyone had any issues with setting up the triggering between the Aria and the Leica scope? We have all the modules and cables, but it seems like the fluidics machine isn't able to send the signal to the scope to start imaging. Any tips/advice would be appreciated!

**Comment by radtkea** [comment]
Hi Ryan!

Thanks for asking your question in the open. I haven't but perhaps @colinchu72 can comment? Colin could you also connect Ryan to your people at Fluigent?

Best wishes and keep us posted!
Andrea

**Comment by colinchu72** [comment]
Hi Ryan, we have the Aria and Leica Thunder scope. We use the TTL to connect and they work well for us. In the Aria software we once missed the 'Bell' symbol toggle that triggers the signal out to the microscope at the end of the step. Is that the case? Happy to follow up separately if more input needed!
